contracts/CertiKSecurityOracle.sol

```diff
@@ -23,11 +23,51 @@ contract CertiKSecurityOracle is Ownable {
   mapping(address => mapping(bytes4 => Result)) private _results;
   // score to return when we don't have results available
   uint8 public defaultScore;
+  // set permitted contribuer role
+  bytes32 public constant CONTRIBUTOR_ROLE = keccak256("CONTRIBUTOR_ROLE");
```

Comment:
if follow github convention then external developers being invited to a project is called "collaborator", could this be a better name?

Reply:
Fixed. Thanks for the comment